{"timestamp":"2025-10-26T02:01:02.144559Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-26T02:01:02.146152Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/test_postgres_dag.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-26T02:01:02.185928Z","level":"warning","event":"The `airflow.operators.python.PythonOperator` attribute is deprecated. Please use `'airflow.providers.standard.operators.python.PythonOperator'`.","category":"DeprecatedImportWarning","filename":"/opt/airflow/dags/test_postgres_dag.py","lineno":2,"logger":"py.warnings"}
{"timestamp":"2025-10-26T02:01:02.206717Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.207380Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.207954Z","level":"info","event":"Current task name:check_postgres_connection","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.208429Z","level":"info","event":"Dag name:test_postgres_dag","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.219644Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":972,"error_detail":[{"exc_type":"AirflowNotFoundException","exc_value":"The conn_id `postgres_docker` isn't defined","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":82,"name":"run"},{"filename":"/opt/airflow/dags/test_postgres_dag.py","lineno":14,"name":"test_postgres_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py","lineno":234,"name":"get_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":243,"name":"connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/hook.py","lineno":61,"name":"get_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/connection.py","lineno":228,"name":"get"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/connection.py","lineno":218,"name":"_handle_connection_error"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-26T02:01:02.221678Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.222281Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.222717Z","level":"info","event":"Task:<Task(PythonOperator): check_postgres_connection>","logger":"task.stdout"}
{"timestamp":"2025-10-26T02:01:02.223090Z","level":"info","event":"Failure caused by The conn_id `postgres_docker` isn't defined","logger":"task.stdout"}
